test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow.sh

```diff
@@ -34,11 +34,11 @@ install_kubeflow(){
 # Function to remove kubeflow in EKS cluster using kustomize
 uninstall_kubeflow(){
 
+    echo "> Uninstalling training operators"
```

Comment:
Why does the ordering matter for kubeflow whether it uninstalls operators first or the namespace?

Reply:
Deleting namespaces first, removes information that we need to uninstall the operators. It basically leads to endless loop scenario.

Reply:
how was the testing done earlier if this does not work?